tests/test_c_code/test_basic.c

```diff
@@ -0,0 +1,619 @@
+/* Include the generated files first to test that all required header
+   files are included. */
+#include "motohawk.h"
+#include "padding_bit_order.h"
+#include "vehicle.h"
+#include "open_actuator.h"
+#include "multiplex.h"
+#include "multiplex_2.h"
+#include "floating_point.h"
+#include "no_signals.h"
+#include "signed.h"
+#include "my_database_name.h"
+#include "min_max_only_6_0.h"
+#include "abs.h"
+
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <math.h>
+#include "unity.h"
```

Comment:
IMO, that header name has quite a bit potential for confusion. is is possible to include it from a sub-directory a la `unity-unit-test-framework/unity.h`